packages/renderer/src/min-max.ts

```diff
@@ -0,0 +1,34 @@
+// Standard library Math.min and Math.max can throw
+// if array length is very long. Fixing this with own implementation
+
+export const min = (arr: number[]) => {
+	if (arr.length === 0) {
+		throw new Error('Array of 0 length');
```

Comment:
when `arr.length` is 0, it throws error and it's causing CI to fail. I guess best way to handle this condition is to return `0` here cause `Math.min([]) = 0`.

Reply:
This would actually lead to some unintended evaluation - if you would render the audio, it would lead to `numberLength === "Infinity"`. This was a bug but it had no negative consequences. But thanks to the test we noticed it and I managed to improve upon it now 😇